README.md

```diff
@@ -60,6 +60,14 @@ This is used for deploying locally, it automatically generates one for each devi
 
 You may notice that after cloning the repo that you are missing a docker-compose.yml file which is required to push the containers to balena. This is a result of us generating the docker-compose file based on a settings.ini file in the root directory. You can also find the latest docker-compose files used in production in the [device-compose-files folder](https://github.com/NebraLtd/helium-miner-software/tree/master/device-compose-files) (generated via github workflow). Here are the steps of generating a new docker-compose.yml file:
 
+*Note for generating the file for local tests: The repo's short Git SHA is fetched from GitHub pipeline automatically while woorking with standard procedure. But if you need to test your working copy locally, you need to create an environment variable first via running following command on Linux:*
```

Comment:
Very minor but there's a typo on working here... Two Os instead of 1

Reply:
Oops! :sweat_smile:

Reply:
Fixed and amended into previous commit.